ui/components/MesheryCard.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from "react";
+import {
+  Avatar,
+  Divider,
+  Grid,
+  IconButton,
+  Typography,
+  Box,
+  Button
+} from "@mui/material";
+import { styled } from "@mui/material/styles";
+import DoneAllIcon from "@mui/icons-material/DoneAll";
+import RemoveDoneIcon from "@mui/icons-material/RemoveDone";
+import FullscreenIcon from "@mui/icons-material/Fullscreen";
+import FullscreenExitIcon from "@mui/icons-material/FullscreenExit";
+import DeleteIcon from "@mui/icons-material/Delete";
+import SaveIcon from "@mui/icons-material/Save";
+import FlipCard from "./FlipCard"
+const ActionContainer = styled(Box)(({ theme }) => ({
+  display: "flex",
+  gap:theme.spacing(1),
+  justifyContent:"flex-end"
+}));
+const INITIAL_GRID_SIZE = { xl : 4, md : 6, xs : 12 };
+const MesheryCard = ({
+  name,
+  updated_at,
+  created_at,
+  file,
+  type,
+  handleDeploy,
+  handleUnDeploy,
+  updateHandler,
+  deleteHandler,
+  setSelectedPatterns,
```

Comment:
@harkiratsm  Instead of adding `setSelectedPatterns `props, why don't you add a prop named `configurations` like in the `uploadimport.js ` and add a if statement for the configuration named design

Reply:
I guess for "file" , the name should be configuration_files.. for better understanding

Reply:
We are not passing any file , we are just specifying the configuration where the cards are used so `configurations` is a better option

Reply:
for the ```type`` I changed that to the ```configurations``` , for the better understanding

Reply:
@YashKamboj ```setSelectedPatterns``` is the passed as props from the patterns .